spec/rubocop/cop/rails/uniq_before_pluck_spec.rb

```diff
@@ -3,55 +3,83 @@
 
 require 'spec_helper'
 
-describe RuboCop::Cop::Rails::UniqBeforePluck do
-  let(:corrected) { 'Model.uniq.pluck(:id)' }
-  subject(:cop) { described_class.new }
+describe RuboCop::Cop::Rails::UniqBeforePluck, :config do
+  subject(:cop) { described_class.new(config) }
 
-  shared_examples_for 'UniqBeforePluck cop' do |source, action|
+  shared_examples_for 'UniqBeforePluck cop' do |method, source, action|
     if action == :correct
```

Comment:
The action should be called `:report` if we don't auto-correct. But I wonder if it's really necessary to remove auto-correct now that we have a conservative mode. I said in #3148 that it might be necessary to remove it, but then again, it might not be necessary. :smile: What do you think?